src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -343,4 +347,16 @@ export class CasesService {
           });
     }
   }
+
+  /*
+  Checks if the user has role of pui-case-manager and returns true or false
+  */
+  private isPuiCaseManager(): boolean {
+    const userInfoStr = this.sessionStorageService.getItem('userDetails');
+    if (userInfoStr) {
+      const userInfo: UserInfo = JSON.parse(userInfoStr);
+      return userInfo && userInfo.roles && userInfo.roles.indexOf(CasesService.PUI_CASE_MANAGER) !== -1
```

Comment:
This line is dodgy - logical && has higher precedence that strict inequality, so I think this means 
(userInfo && userInfo.roles && userInfo.roles.indexOf(CasesService.PUI_CASE_MANAGER)) !== -1 when it should be 
userInfo && userInfo.roles && (userInfo.roles.indexOf(CasesService.PUI_CASE_MANAGER) !== -1)
Since nobody can ever remember the precedence table, it's always best to have the brackets

Reply:
ok, good point, will rectify